cnd/src/main.rs

```diff
@@ -178,11 +193,19 @@ fn dump_config(settings: Settings) -> anyhow::Result<()> {
 }
 
 /// Throws an error if the connected connected blockchain network is invalid
-fn handle_invalid_network_validation<T: std::fmt::Debug>(validation: NetworkValidationResult<T>) -> anyhow::Result<(), ConfigValidationError<T>> {
+fn handle_invalid_network_validation<T: std::fmt::Debug>(
```

Comment:
What is going on here?
Are we not validating rustfmt in CI anymore or why is this changed?

Reply:
This was a tooling issue, resolved by: https://github.com/comit-network/comit-rs/pull/2218